src/Template/Element/Layout/default/footer.ctp

```diff
@@ -29,7 +29,7 @@
         <div class="row">
             <div class="col-md-12 text-center mt30">
                 <a href="https://cakefoundation.org" target="_blank"><span class="glyph_range icon-expertise buss-icon-8">u</span></a>
-                <p class="copyright">&copy; <?= __('Copyright 2005-2016 Cake Software Foundation, Inc. All rights reserved. {0}.',
+                <p class="copyright">&copy; <?= __('Copyright 2005-2021 Cake Software Foundation, Inc. All rights reserved. {0}.',
```

Comment:
I don't think we need the copyright year.  Just use the same as code:

> Copyright (c) Cake Software Foundation, Inc.

Reply:
Thank you, I will do